runtime/core/portable_type/c10/c10/util/irange.h

```diff
@@ -0,0 +1,123 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#pragma once
```

Comment:
If this file is copied over, can you add a comment here to the effect?

Reply:
> can you add a comment

no. the file should be identical to the version in pytorch core.